roles/osp_observability/templates/prometheus.yml.j2

```diff
@@ -1,18 +1,34 @@
-#jinja2: trim_blocks:False
 global:
   scrape_interval: {{ prometheus_scrape_interval }}
   external_labels:
-    {% if ansible_fqdn %}agent_host: {{ ansible_fqdn }}{% endif %}
-    {% if openstack_cloud_name %}cloud_name: {{ openstack_cloud_name }}{% endif %}
+    {% if ansible_fqdn -%}
+    agent_host: {{ ansible_fqdn }}
+    {% endif -%}
+    {% if openstack_cloud_name -%}
```

Comment:
Same comment as above about checking for definition of variable.

Reply:
https://github.com/infrawatch/osp-observability-ansible/pull/19/files#diff-7027c90a62527124195b5351e33095a6077349eaac7e013201e80bb567d2dcb8R2